x-pack/plugins/spaces/public/nav_control/solution_view_tour/solution_view_tour.tsx

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButtonEmpty, EuiLink, EuiText, EuiTourStep } from '@elastic/eui';
+import React from 'react';
+import type { FC, PropsWithChildren } from 'react';
+
+import type { OnBoardingDefaultSolution } from '@kbn/cloud-plugin/common';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+import type { SolutionView } from '../../../common';
+
+const tourLearnMoreLink = 'https://ela.st/left-nav';
+
+const LearnMoreLink = () => (
+  <EuiLink href={tourLearnMoreLink} target="_blank" external>
+    {i18n.translate('xpack.spaces.navControl.tour.learnMore', {
+      defaultMessage: 'Learn more',
+    })}
+  </EuiLink>
+);
+
+const solutionMap: Record<OnBoardingDefaultSolution, string> = {
+  es: i18n.translate('xpack.spaces.navControl.tour.esSolution', {
+    defaultMessage: 'Search',
+  }),
+  security: i18n.translate('xpack.spaces.navControl.tour.securitySolution', {
+    defaultMessage: 'Security',
+  }),
+  oblt: i18n.translate('xpack.spaces.navControl.tour.obltSolution', {
+    defaultMessage: 'Observability',
+  }),
+};
+
+interface Props extends PropsWithChildren<{}> {
+  solution?: SolutionView;
+  isTourOpen: boolean;
+  onFinishTour: () => void;
+}
+
+export const SolutionViewTour: FC<Props> = ({
+  children,
+  solution,
+  isTourOpen: _isTourOpen,
+  onFinishTour,
+}) => {
+  const solutionLabel = solution && solution !== 'classic' ? solutionMap[solution] : '';
```

Comment:
__question:__ Is it possible to early-return `children` if `solutionLabel` is empty - otherwise it's a bit confusing since all i18n labels below are wrong with empty solution.

Reply:
Addressed in https://github.com/elastic/kibana/pull/194926/commits/7d2d8a768f00430f744056e27b1a1366cd6aab42